tareas/tarea-6/tarea-clase-6-k2.html

```diff
@@ -0,0 +1,150 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>K</title>
+</head>
+<style>
+    .botones{
+        display: flex;
+        justify-content: center;
+    }
+    .botones > input{
+        margin: 10px;
+    }
+    #divisores{
+        display: flex;
+    }
+    #todo{
+        text-align:center;
+        float: right;
+    }
+    .Datos{
+        float: left;
+    }
+    ul > *{
+        justify-content: center;
+        text-align:center
+    }
+</style>
+<body>
+    <div id="divisor">
+        <div class="Datos">
+            <h1>El que mas gana: </h1>
+            <h2>El promedio anul es : </h2>
+            <h2>El promedio mensual es : </h2>
+            <h3>El que menos gana: </h3>
+        </div>
+        <form id="todo">
+            <ul id="#formulario">
+                <label>Integrantes de la family</label>
+                <input id="#family" type="number" min ="0">
+            </ul>
+        <div class="botones">
+            <input type="button" id="submit" value="Calcular">
+            <input type="button" id="delete-this" value="Borra todo">
+        </div>
+        </form>
+    </div>
+</body>
+<script>
+    let cantidadVieja = 0;
+    let cantidadActual = 0;
+    const integrantes = document.getElementById("#family");
+    const borrarTodo = document.getElementById("delete-this");
+    const calcular = document.getElementById("submit"); 
+    borrarTodo.addEventListener("click",function(e){
+        let i = 0;
+        while(cantidadActual>=i){ 
+                del();
+                cantidadActual--;
+        }
+        integrantes.value = 0;
+        modify_data("","","");
+    })
+    integrantes.addEventListener("change",function(e){
+        cantidadActual = integrantes.value;
+        if(cantidadActual>cantidadVieja){
+            while(cantidadVieja!=cantidadActual){ 
+                add();
+                cantidadVieja++;
+            }
+        }else{
+            while(cantidadVieja!=cantidadActual){ 
+                del();
+                cantidadVieja--;
+            }
+        } 
+    })
+
+    function add(){
+        const l = document.createElement("label");
+        const text = document.createTextNode("Cuanto gana anual?");
+        l.appendChild(text);
+        l.style = "display:block";
+        const i = document.createElement("input");
+        i.style = "display:block , justify-content: center";
+        i.className = "edades";
+        i.type = "number";
+        document.getElementById("#formulario").appendChild(l);
+        document.getElementById("#formulario").appendChild(i);
+    }
+    function del(){
+        const e = document.getElementById("#formulario"); 
+        if(e.children.length>2){
+            e.removeChild(e.lastChild);
+            e.removeChild(e.lastChild);
+        }
+    }
+
+    calcular.addEventListener("click",function(e){
+        const total = document.getElementsByClassName("edades");
+        let suma = 0;
+        let min = 9999;
```

Comment:
```suggestion
        let salarioMinimo = 9999;
```

No es una buena estrategia ponerle un número por default, mejor dejalo `undefined`o ponele `null` y preguntá por eso adentro del `for`. En este caso, si todos cobran más de 10 lucas, el salario mínimo va a ser 9999.

Reply:
esto se me pasó, lo usaba para calcular la edad, pero era porque no tenia idea de como hacer para comparar, ahora que se que funciona voy a usar null para comparar por un menor